library/src/webapp/editor/ckextraplugins/common/css/dialog.css

```diff
@@ -0,0 +1,27 @@
+/* search results paging styles */
```

Comment:
It's just that normally if you're wanting to share styling across multiple nodes in different contexts you'd apply a class to them.